lib/host-sp-messages/src/lib.rs

```diff
@@ -507,10 +507,38 @@ impl From<HubpackError> for DecodeFailureReason {
     }
 }
 
+#[derive(
+    Copy,
+    Clone,
+    Debug,
+    Eq,
+    PartialEq,
+    Serialize,
+    Deserialize,
+    SerializedSize,
+    FromBytes,
+    AsBytes,
+)]
+#[repr(transparent)]
+pub struct Status(u64);
+
+#[derive(
+    Copy,
+    Clone,
+    Debug,
+    Eq,
+    PartialEq,
+    Serialize,
+    Deserialize,
+    SerializedSize,
+    FromBytes,
+    AsBytes,
+)]
+#[repr(transparent)]
+pub struct HostStartupOptions(u64);
```

Comment:
Did these get moved out of the `bitflags` macro for the same reason as below or another reason?

Reply:
Any case where I pulled structs out of the bitflags macro, it's because they needed to `derive` a trait that doesn't automatically work on the generated types for structs defined _inside_ the 2.x macro. This was most of the annoying work.